game/scripts/npc/items/item_revenants_brooch.txt

```diff
@@ -65,14 +65,14 @@
     {
       "bonus_intellect"   "45 50 60 75 95"
       "bonus_attack_speed"  "40 45 50 55 60"  
-      "bonus_armor"     "8 9 11 14 18"
+      "bonus_armor"     "8 9 10 11 12"
```

Comment:
I am not a fan of the bad armor scaling. If you want to nerf it, then nerf the first lvl as well while preserving the scaling. Like this: 6/7/9/12/16 (Witch Blade gives 6 armor)